src/ensembl/src/shared/components/help-article/VideoArticle.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useEffect, useRef } from 'react';
+
+import useResizeObserver from 'src/shared/hooks/useResizeObserver';
+
+import { CircleLoader } from 'src/shared/components/loader/Loader';
+
+import { LoadingState } from 'src/shared/types/loading-state';
+import { VideoArticle as VideoArticleType } from 'src/shared/components/help-popup/types';
```

Comment:
Should this be renamed to `VideoArticleData`?

Reply:
Yeah, and the types in the `src/shared/components/help-popup/types` have been moved to a different file; so I cleaned up the code there as well.